Version Packages (#2774)
jpmorganchase/salt-ds: packages/ag-grid-theme/CHANGELOG.md

# @salt-ds/ag-grid-theme

## 1.3.6

### Patch Changes

> - cdf44770: Fix border misalignment on range selection when a sigle column or a single row is selected.

Reviewer: ```suggestion
- cdf44770: Fixed border misalignment in range selection on a single column or a single row.
```
